test/integration/cli-volumes.test.js

```diff
@@ -169,4 +177,73 @@ test('triton volume create ...', testOpts, function (tt) {
             });
     });
 
+    // Test that we can create a volume with a valid fabric network and the
+    // volume ends up on that network.
+
+    tt.test('  find fabric network', function (t) {
+        h.triton(['network', 'list', '-j'].join(' '),
+            function onGetNetworks(getNetworksErr, stdout, stderr) {
+                var resultsObj;
+
+                t.ifErr(getNetworksErr, 'should succeed getting network list');
+
+                // turn the JSON lines into a JSON object
+                resultsObj = JSON.parse('[' + stdout.trim().replace(/\n/g, ',')
+                    + ']');
+
+                t.ok(resultsObj.length > 0,
+                    'should find at least 1 network, found '
+                    + resultsObj.length);
+
+                FABRIC_NETWORKS = resultsObj.filter(function fabricFilter(net) {
+                    // keep only those networks that are marked as fabric=true
+                    return (net.fabric === true);
+                });
+
+                t.ok(FABRIC_NETWORKS.length > 0,
+                    'should find at least 1 fabric network, found '
+                    + FABRIC_NETWORKS.length);
+
+                t.end();
+            });
+    });
+
+    tt.test('  triton volume on fabric network', function (t) {
+        h.triton([
```

Comment:
Would it be helpful to give that volume a name like `node-triton-test-volume-create-fabric-network`, so that when for some reason the test fails, or the volume ends up not being deleted we can find out more easily what created it?